src/lib/entities/podcast/ui/podcast-tile.svelte

```diff
@@ -24,11 +23,14 @@
   )}
   href="/podcasts/{podcast.id}"
 >
-  <img
-    class="rounded-lg aspect-square"
-    width={imageSize}
-    height={imageSize}
-    class:row-span-2={!large}
+  <Image
+    class={cx(
+      'rounded-lg',
+      'aspect-square',
+      'overflow-hidden',
+      large ? 'w-27 h-27' : 'w-8 h-8',
+      !large && 'row-span-2'
```

Comment:
Why not combine into one?
```suggestion
      large ? 'w-27 h-27' : 'w-8 h-8 row-span-2',
```

Reply:
They are different in nature, I think it aids readability to keep them separate